DXRModules/DeusEx/Classes/DXRPasswords.uc

```diff
@@ -272,14 +273,20 @@ function RandoInfoDevs(int percent)
         if( id.plaintext != "" ) {
             ProcessStringHasPass(id.plaintext, hasPass);
         }
-        i=0;
+
         num=0;
         foreach AllActors(class'Inventory', inv)
         {
-            if( SkipActor(inv, 'Inventory') ) continue;
+            if( SkipActorBase(inv) ) continue;
```

Comment:
actually SkipActorBase doesn't check _skipactor_types, so I need to fix this or else it'll accept NanoKeys too